package.json

```diff
@@ -778,14 +778,14 @@
         "prepare": "husky install"
     },
     "devDependencies": {
-        "@microsoft/vscode-azext-dev": "^2.0.1",
         "@microsoft/eslint-config-azuretools": "^0.2.1",
+        "@microsoft/vscode-azext-dev": "^2.0.1",
```

Comment:
Any idea why this was out of order to begin with? lol

Reply:
No idea, pretty weird.